node/node.go

```diff
@@ -347,7 +385,7 @@ func (n *Node) initNetworking(primaryNetVdrs validators.Set) error {
 		n.MetricsRegisterer,
 		n.Log,
 		listener,
-		dialer.NewDialer(constants.NetworkType, n.Config.NetworkConfig.DialerConfig, n.Log),
+		dialer.NewDialer(networkType, n.Config.NetworkConfig.DialerConfig, n.Log),
```

Comment:
The dialer should not be specific to `tcp4` or `tcp6` based on the local node's IP (so I think this should always be `tcp`.)